.travis.yml

```diff
@@ -171,7 +171,7 @@ before_install:
 
       git clone https://github.com/philsquared/Catch.git;
       cd Catch;
-      git checkout v1.9.4;
+      git checkout v2.1.0;
```

Comment:
I'm testing the changes to support catch 2.1.2. Once that is rebased we should we should bump this to v2.1.2 instead of 2.1.0 so that 2.1.2 and newer remains supported in the future.